internal/services/zero_trust_network_hostname_route/testdata/basic.tf

```diff
@@ -0,0 +1,17 @@
+resource "cloudflare_zero_trust_tunnel_cloudflared" "%[1]s" {
+  account_id    = "%[2]s"
+  name          = "%[1]s"
+  tunnel_secret = "AQIDBAUGBwgBAgMEBQYHCAECAwQFBgcIAQIDBAUGBwg="
```

Comment:
Is this a secret for an actual tunnel?

Reply:
You can generate the secret in the test, randomly since no one will connect to this tunnel.